modules/constructor/constructor-bottom/constructor-bottom.service.js

```diff
@@ -1,12 +1,9 @@
 const ConstructorBottom = require('./constructor-bottom.model');
-const Material = require('../../material/material.service');
 const {
   CONSTRUCTOR_BOTTOM_NOT_FOUND,
-  IMAGE_NOT_FOUND,
-  CONSTRUCTOR_BOTTOM_ALREADY_EXIST,
+  CONSTRUCTOR_BOTTOM_ALREADY_EXIST
 } = require('../../../error-messages/constructor-bottom.messages');
-const uploadService = require('../../upload/upload.service');
-const { calculatePrice } = require('../../currency/currency.utils');
+const {calculatePrice} = require('../../../utils/calculate-price');
```

Comment:
We already have this in `currency/currency.utils`

Reply:
Fixed